src/eval-frame/actions/actions.js

```diff
@@ -73,11 +73,12 @@ export function appendToEvalHistory(cellId, content, value, historyOptions = {})
   }
 }
 
-export function updateValueInHistory(historyId, value) {
+export function updateValueInHistory(historyId, value, historyType) {
   return {
     type: 'UPDATE_VALUE_IN_HISTORY',
     historyId,
     value,
+    historyType,
```

Comment:
i don't think we really need to add functionality to change historyTypes, see other comments